secrethub_wrapper.go

```diff
@@ -45,7 +46,7 @@ import (
 
 var clients sync.Map
 var clientIDMutex sync.Mutex
-var nextClientID = 1;
+var nextClientID uint64 = 1
```

Comment:
Not really relevant, but just out of curiosity: is there a reason we're starting at `1`?

Reply:
No particular reason, but we can change this any time.

Reply:
Okay, I understand. So are you also responsible for the 1-based indexing in MATLAB and R?